### DISCUSSION TITLE
blog/rubbish-liyupi

### DISCUSSION THREAD

**Discussion by giscus** [jsun969/jsundotlol#5] [Announcements]
# blog/rubbish-liyupi

❤❤❤永远守护最好的鱼皮大大❤❤❤

https://jsun.lol/blog/rubbish-liyupi

**Comment by mazhichen8780** [comment]
你不要以为大学生就可以为所欲为，你早就被曝光了：
https://mp.weixin.qq.com/s/z9V2EDi0kkl34f4d4CACDQ
那篇文章是看在你无知的情况下删的!

**Reply by jsun969** [reply]
我高中生。。

**Comment by mazhichen8780** [comment]
在做黑子的道路上，你真的是 Number One，必须给你发个奖状【功德减亿奖】！

**Reply by jsun969** [reply]
谢谢！

**Comment by mazhichen8780** [comment]
如果你觉得鱼皮做的星球里的内容简单，那只是说明你并不是星球的受众，星球里的内容对你没用而已，你可以选择不加入，但不要带着刻板的印象一棒子打死知识付费或去当键盘侠。

星球本身的价格就是一节付费课的价格，如果有人加入星球后，能学到知识、能得到改变，对他来说，就有价值，不是么？你这样只会显得你很弱智

**Reply by jsun969** [reply]
好的！

**Comment by mazhichen8780** [comment]
因为鱼皮赚钱了，赚钱了就有像你这样的键盘侠眼红，眼红了就会难受，难受了就会匿名发帖 “如何评价B站【程序员鱼皮】” 来发泄情绪。因为在你们（傻子）眼里，鱼皮不配赚这个钱！你们这些键盘侠就觉得鱼皮在嘎韭菜！咔咔割~

割韭菜，何为割韭菜？
你们这些喷子，没文化，真可怕

**Reply by jsun969** [reply]
这问题还真不是我匿名提问的，不好意思

**Comment by mazhichen8780** [comment]
还敢发邮箱：
[jsun969/jsundotlol] blog/rubbish-liyupi (Discussion #5)
blog/rubbish-liyupi
❤❤❤永远守护最好的鱼皮大大❤❤❤
https://jsun.lol/blog/rubbish-liyupi

**Reply by jsun969** [reply]
这是GitHub自动回复的，哥。。

**Comment by cyeonix** [comment]
同样反感+1，没看出来技术怎么样，但是嘴皮子和忽悠程度确实不错-_-

**Comment by mazhichen8780** [comment]
你咋不讲讲鱼皮自建公司呢



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年07月30日 20:38 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [jsun969/jsundotlol] blog/rubbish-liyupi (Discussion #5) |

同样反感+1，没看出来技术怎么样，但是嘴皮子和忽悠程度确实不错-_-

**Comment by lcxskylake** [comment]
确实水，会包装，善营销，不知道买了多少推广，又迎合了当下的程序员焦虑，专心搞技术不如捞偏门

**Comment by zhoupingxiao** [comment]
看不了那张嘴，永远合不上的感觉。😂
